v4/source/reactions.yaml

```diff
@@ -89,3 +89,33 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+
+  '/posts/ids/reactions':
+    post:
+      tags:
+        - reactions
+      summary: Bulk get the reaction for posts
+      description: |
+        Get a list of reactions made by all users to a given post.
+        ##### Permissions
+        Must have `read_channel` permission for the channel the post is in.
```

Comment:
Can you add a minimum server version of 5.8 here? You can see how other endpoints do it as an example